rasa_core/processor.py

```diff
@@ -287,10 +287,24 @@ def _run_action(self, action, tracker, dispatcher):
             logger.error(e, exc_info=True)
             events = []
         self._log_action_on_tracker(tracker, action.name(), events)
+        self._log_bot_utterances_on_tracker(tracker, dispatcher)
```

Comment:
let's do this before logging the actions (makes a bit more sense, e.g. if the action created the restart event, that one would now be logged before the bot utterances - it makes a bit more sense the other way around).